backend/inspirehep/utils.py

```diff
@@ -85,6 +85,21 @@ def get_failure_message_for_index_record(task_name, exception, **kwargs):
     return format_failure_message_for_single(task_name, exception, kwargs.get("uuid"))
 
 
+def get_failure_message_for_disambiguate_records(task_name, exception, *args, **kwargs):
+    if isinstance(args[0], list):
+        return format_failure_message_for_multiple(task_name, exception, args[0])
+    return format_failure_message_for_single(task_name, exception, args[0])
+
+
+def get_failure_message_for_celery_starmap(task_name, exception, *args, **kwargs):
+    task_name = kwargs.get("task")["task"]
```

Comment:
if task is not present, `get('task')` will return `None`, and by doing `["task"]` will throw a `TypeError`. You can make it that part fail safe and only proceed if task name exists